### ISSUE TITLE
identify user work-phases with rough wireframes.  changes to views not part of this

### ISSUE THREAD

**Issue by kundanbapat** [kundanbapat/JSON-samples-factory#2]
Tentative phases:
*Phase 1*:
get JSON content from user; validate, format, and save it.

*Phase 2*:
Read sample JSON to identify type syntax (string, number, null, boolean, array, object).  Generate form for user to create new JSON - pre-populate with values from the original sample (which can be overwritten or accepted as is).  

For array element at top level or nested at any level:
- Pre-populate with all key/value pairs from user sample.
- Allow addition of new elements.
- Allow deletion of pre-existing or newly added elements.
- Allow elements in array to be ANY type. NO constraints on types inside array in MVP.
- DO validate syntax of newly entered value - which could be string, number, null, array or object literal.

For JSON values of type object:
- Addition of new keys is NOT supported.
- Deletion of existing keys is NOT supported.
- Names of keys MAY NOT be changed.
- Values can be edited - but the syntax type of values stay the same as the user sample.

*Phase 3*:
NOT for MVP.  after phase 2 is updated AFTER MVP to let user specify number ranges, string enums, specify random numbers, and number of samples etc. - this phase generates those many new samples at random, based on these user-specified criteria.
